lib/datadog/tracing/contrib/propagation/sql_comment/ext.rb

```diff
@@ -0,0 +1,21 @@
+# typed: false
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Propagation
+        module SqlComment
+          module Ext
+            ENV_SQL_COMMENT_PROPAGATION_MODE = 'DD_TRACE_SQL_COMMENT_PROPAGATION_MODE'.freeze
+
+            DISABLED = 'disabled'.freeze
+            SERVICE = 'service'.freeze
+            FULL = 'full'.freeze
+
+            TAG_DBM_TRACE_INJECTED = '_dd.dbm_trace_injected'.freeze
```

Comment:
I recommend documenting when this tag is applied, and what value it can be set to.